codegen/src/main/java/io/spine/tools/properties/PropertiesWriter.java

```diff
@@ -54,11 +54,17 @@ public PropertiesWriter(String folder, String fileName) {
     }
 
     /**
-     * Updates the {@code .properties} file rewriting its contents if it already exists.
+     * Updates the content of the file with the passed map.
      *
-     * @param propertiesMap a map containing properties to write to the file
+     * <p>If the file already exists, the passed entries will be merged with the file content.
+     * If the file already has an key from the passed map, its value will <strong>NOT</strong>
```

Comment:
We tend to use `<b>` and `<strong>` for the same goal. Let's settle on `<b>`, as it's shorter.

Reply:
Using `<em>` is more correct in this case as it's a single word in the sentence.